build/Targets/Imports.targets

```diff
@@ -498,4 +498,13 @@
     <AdditionalFiles Include="@(PublicAPI)" />
   </ItemGroup>
 
+  <!-- CPS doesn't show these items by default, but we want to show them. -->
```

Comment:
Looks like @srivatsn added that line here

https://github.com/dotnet/roslyn/commit/245012e434daa4a9ad6ba8c0028086aea6658188

Probably need an item to track undoing that + this once we have the right project system fix.